cirq-google/cirq_google/engine/simulated_local_program.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional, TYPE_CHECKING
+
+from cirq_google.engine.abstract_local_program import AbstractLocalProgram
+from cirq_google.engine.local_simulation_type import LocalSimulationType
+
+if TYPE_CHECKING:
+    from cirq_google.engine.abstract_job import AbstractJob
+    from cirq_google.engine.abstract_engine import AbstractEngine
+    from cirq_google.engine.simulated_local_processor import SimulatedLocalProcessor
+
+
+class SimulatedLocalProgram(AbstractLocalProgram):
```

Comment:
Tests for this class? Kind of surprised coverage check isn't firing for this...

Reply:
Added some basic tests.  Coverage was previously provided by tests in processor and engine.

Reply:
Where are these tests? There's no `simulated_local_program_test.py` in this PR - did it get missed in the commit?